platform/kobo/koreader.sh

```diff
@@ -241,6 +241,36 @@ if [ -z "${INTERFACE}" ]; then
     export INTERFACE
 fi
 
+# We'll enforce UR in ko_do_fbdepth, so make sure further FBInk usage (USBMS)
+# will also enforce UR... (Only actually meaningful on sunxi).
+if [ "${PLATFORM}" = "b300-ntx" ]; then
+    export FBINK_FORCE_ROTA=0
+    # And we also cannot use batched updates for the crash screens, as buffers are private,
+    # so each invocation essentially draws in a different buffer...
+    FBINK_BATCH_FLAG=""
+    # Same idea for backgroundless...
+    FBINK_BGLESS_FLAG="-B GRAY9"
+    # It also means we need explicit background padding in the OT codepath...
+    FBINK_OT_PADDING=",padding=BOTH"
+
+    # Make sure we poke the right input device
+    KOBO_TS_INPUT="/dev/input/by-path/platform-0-0010-event"
+else
+    FBINK_BATCH_FLAG="-b"
+    FBINK_BGLESS_FLAG="-O"
+    FBINK_OT_PADDING=""
+    KOBO_TS_INPUT="/dev/input/event1"
+fi
+
+# Make sure we only have only two cores online on the Elipsa.
+# NOTE: That's a bit optimistic, we might actually need to tone that down to one,
+#       and just online the second one on demand (e.g., PDF).
+if [ "${PRODUCT}" = "europa" ]; then
+    echo "1" >"/sys/devices/system/cpu/cpu1/online"
+    echo "0" >"/sys/devices/system/cpu/cpu2/online"
+    echo "0" >"/sys/devices/system/cpu/cpu3/online"
```

Comment:
>  `# Make sure we only have only two cores online `

But you _did_ tore that down to one ? Or echo "1" means disable ?
(Will cpu1 core suffers if it's the only one getting used ? :)

Reply:
CPUs start counting at 0, and you can't offline CPU0 (for obvious reasons ;p).

Reply:
Ok :) (Too much Lua and counting from 1!)